connectors/opa/go.mod

```diff
@@ -8,7 +8,7 @@ go 1.13
 
 require (
 	github.com/golang/protobuf v1.4.2
-	github.com/hashicorp/go-retryablehttp v0.5.4
+	github.com/hashicorp/go-retryablehttp v0.6.6
```

Comment:
agree, this should not be here, this was a result of building the opa connector previously, which indicates that the `go.mod` there is stale, but going forward we should probably eliminate the private go.mod and use the top-level go.mod. in any case will remove this change from this PR

Reply:
this is a result of build/deploying opa connector, its' a result of having stale go.mod dependencies in opa and elsewhere, so note that if I remove this (and other connector go.mod changes I introduced) then each time the user will run the test he will get all these changes locally ... which is not what we want ... we want the code to be updated so that anyone running the test will not trigger additional changes to the code base.